src/ray/core_worker/core_worker.cc

```diff
@@ -1169,6 +1174,14 @@ Status CoreWorker::SetResource(const std::string &resource_name, const double ca
   return local_raylet_client_->SetResource(resource_name, capacity, client_id);
 }
 
+Status CoreWorker::ForceSpillObjects(const std::vector<ObjectID> &object_ids) {
+  return local_raylet_client_->ForceSpillObjects(object_ids);
```

Comment:
Won't this only work if the master copy is on the local machine?

Reply:
yes, but the intention of this API (`force_spill_objects`) is a simple proof of object spilling (make sure I/O workers and other interfaces work properly), and it will only be used in tests or some temporary hotfix. It is clear that we won't build queries for the location of the master copy for this interface.